build/release.sh

```diff
@@ -38,4 +38,10 @@ fi
 
 kube::build::copy_output
 kube::release::package_tarballs
+# If we have these variables set then we want to build all docker images.
+# Since regular build functions above don't build hyperkube we need to do
+# it manually.
+if [[ -n "${KUBE_DOCKER_IMAGE_TAG-}" && -n "${KUBE_DOCKER_REGISTRY-}" ]]; then
```

Comment:
Maybe my comment wasn't clear.  I agree with @david-mcmahon; this should have a different home.